app/src/test/java/com/ucmobiledevelopment/freeways/IncidentUnitTest.kt

```diff
@@ -87,7 +119,7 @@ class IncidentUnitTest {
         assertTrue(allIncidents!!.isNotEmpty())
         var containsIncidentWithCaseId400132 = false
         allIncidents!!.forEach {
-            if(it.caseId.equals("400132")){
+            if (it.caseId == "400132") {
```

Comment:
We are comparing the actual data for structural equality. https://agrawalsuneet.github.io/blogs/equality-in-kotlin/#:~:text=Referential%20equality%20%28%E2%80%98%3D%3D%3D%E2%80%99%29%20%3D%3D%3Doperator%20is%20used%20to%20compare,the%20values%20are%20not%20equal%20to%20each%20other.?msclkid=2df09cd9af1111eca7c1a300ad8d2350